subcommittees/mentoring/mission-objectives.md

```diff
@@ -0,0 +1,48 @@
+# Mission
+
+The Mentoring Subcommittee was established in 2015 to develop and maintain a mentorship program to support instructors as they progress through training, teaching, curriculum development, and other community-related activities. We help promote community-building and networking by providing (virtual) spaces where instructors from all over the world can share teaching success stories and discuss strategies for overcoming challenges. 
```

Comment:
Why are we splitting this out of the README?  It doesn't seem like there's so much going on here that we need docs in two places.